index.js

```diff
@@ -25,6 +25,10 @@ console.log(readFileSync('bigtitle.txt', 'utf8').toString())
 
 const Switchblade = require('./src/Switchblade.js')
 const client = new Switchblade(CLIENT_OPTIONS)
-client.on('debug', (...args) => console.log('debug', ...args))
-client.on('rateLimit', (...args) => console.log('rateLimit', ...args))
+
+if (process.env.NODE_ENV === 'PRODUCTION') {
+  client.on('debug', (...args) => console.log('debug', ...args))
+  client.on('rateLimit', (...args) => console.log('rateLimit', ...args))
+}
```

Comment:
why?

Reply:
because logging debug logs on development isn't cool, it's too much spam and pretty hard to read the log

Reply:
but if it's a development instance it should print debug logs, I don't think it's too much spam tbh

Reply:
fine